unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -1797,6 +1797,29 @@ createAuthor =
         _ -> Left $ showPatternHelp createAuthor
     )
 
+gist :: InputPattern
+gist =
+  InputPattern
+    "gist"
```

Comment:
What about `push.gist` for grouping and (more importantly) discoverability?

Reply:
We could also do `push.gist` as the main command and `gist` as an alias (line 1804).

Reply:
Okay, I can rename the command to `push.gist` and alias to `gist`.